backend/django/core/templates/projects/list.html

```diff
@@ -9,15 +9,15 @@ <h1>Projects</h1>
     <div class="list-group">
       {% for potential_umbrella_project in object_list %}
         {% if potential_umbrella_project.umbrella_string  %}
-        <div id="umbrella-{{potential_umbrella_project.umbrella_string}}" class="panel panel-default">
+        <div id="umbrella-{{potential_umbrella_project.umbrella_string|slugify}}" class="panel panel-default">
```

Comment:
<img width="1158" alt="Screen Shot 2022-06-13 at 3 21 37 PM" src="https://user-images.githubusercontent.com/10951569/173428898-5c83568f-b40f-45bd-bc56-d5563fadf6f4.png">
So if someone makes a project with only special characters, the issue still remains. Thoughts?

Reply:
Well, the Django docs for [slugify](https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#slugify) say that it removes all characters that are not alphanumerics, underscores, or hyphens and replaces spaces with hyphens. So if there aren't any actual alphanumeric characters then its not going to have anything...

Reply:
Although is it very likely that a user will create a project folder without any alphanumeric characters at all?

Reply:
But if someone does, then the carrot does not work

Reply:
I mean no, it isn't, but it is a known bug so I wouldn't drop it entirely, since "non-alphanumeric" can eventually happen if someone wanted to use non-english characters or something. 

I would suggest we make an issue to require folder names have some alphanumeric character in the creation forms. I don't think this MR needs that to merge, but I'd like to not lose it. @peridoteagle?

Reply:
Ok, I was just wondering if that was something we needed to have functionality for or if it would be ok to simply require user's to have an alphanumeric character.

Reply:
I added a commit to validate the input field has one alphanumeric. We can use the html field `pattern` on the input field to handle this with a regex string.

Reply:
Yes this is working now in the project create page. I think we just need to add it to the update page umbrella.html and we should be good